PDF2text/script.py

```diff
@@ -0,0 +1,15 @@
+import os
+import pdftotext
+
+
+pdf_path = input("Enter the path of the pdf file : ")
+
+assert os.path.exists(pdf_path), "this pdf file doesn't exist"
+
+with open(pdf_path, 'rb') as f_r:
+    pdf_pages = pdftotext.PDF(f_r)
+
+for i, page in enumerate(pdf_pages):
```

Comment:
Save the text of the pages in different text files, one text file per page.

Reply:
> Save the text of the pages in different text files, one text file per page.

Can I know how to do it as I am new to it

Reply:
just save the files using:
```python
with open(f"./PDF2text/pages/{i}.txt", "w") as f:
    f.write(page)
```

I think it should work, do make a local run to confirm